loklak.php

```diff
@@ -55,5 +56,54 @@ public function status() {
 		return json_encode($request, true);
 	}
 
+	public function search($query, $since=null, $until=null, $from_user=null, $count=null){
+        $this->requestURL = $this->baseUrl . '/api/search.json';
+        $this->query = $query;
+        $this->since = $since;
+        $this->until = $until;
+        $this->from_user = $from_user;
+        $this->count = $count;
+        if($query){
+            $params = array('q'=>$this->query);
+            //$params['query'] = $this->$query;
+            if ($since)
+            	$params['q'] = $params['q'] . ' since:'.$this->since;
+         	if ($until)
+         		$params['q'] = $params['q'] . ' until:'.$this->until;
+         	if ($from_user)
+         		$params['q'] = $params['q'] . ' from:'.$this->from_user;
+         	if ($count)
+        		$params['count'] = $this->count;
```

Comment:
Indentations of `if ($since)` etc.., are not proper, these are separate if statements and shouldn't be nested.